DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -129,6 +129,10 @@ public function load(array $configs, ContainerBuilder $container): void
             $container->setParameter('ekino.new_relic.application_name', $config['application_name']);
             $container->setAlias('ekino.new_relic.logs_handler', $config['monolog']['service'])->setPublic(false);
         }
+
+        if (!$config['enabled']) {
```

Comment:
@Nyholm do you know why we don't use in this file
```
if ($config['enabled']) {
    if ($config['commands']['enabled']) {
         // ...
    }
    if ($config['exceptions']['enabled']) {
         // ...
    }
    if ($config['monolog']['enabled']) {
         // ...
    }
}
```

or better

```
public function load(array $configs, ContainerBuilder $container): void
{
    $configuration = new Configuration();
    $config = $this->processConfiguration($configuration, $configs);

    if (!$config['enabled']) {
        return;
    }
    // ...
}
````

Reply:
We "disable" the bundle by using the Blackhole interactor. That way we make sure that potential bugs in our listeners are discovered in test and dev.